rcsb/workflow/targets/ProteinTargetSequenceExecutionWorkflow.py

```diff
@@ -182,7 +205,7 @@ def fullWorkflow():
     ptsWf = ProteinTargetSequenceExecutionWorkflow()
     ok = True
     ok = ptsWf.cacheTaxonomy()
-    ok = ptsWf.fetchUniProtTaxonomy()
+    ok = ptsWf.updateUniProtTaxonomy()
```

Comment:
Why not keep `fetchUniProtTaxonomy()`? It looks like the `updateUniProtTaxonomy()` method takes about an hour to run. Is it necessary to build it from scratch every time?

Reply:
The previous way, that file was never getting updated in BuildLocker. updateUniProtTaxonomy is not called from anywhere else that I could find. That was one of the files that had a modified date of before October 2021 in BuildLocker. I'm open to suggestions on a better way to handle this, if you think it's going to be a problem.

Reply:
As we discussed, `fetchUniProtTaxonomy()` does not update the stash and there is no incremental update, unfortunately. So, keeping `updateUniProtTaxonomy()`.